packages/e2e-tests/specs/writing-flow.test.js

```diff
@@ -34,7 +34,7 @@ describe( 'adding blocks', () => {
 		await page.keyboard.type( 'Paragraph' );
 		await pressKeyTimes( 'Tab', 3 ); // Tab to paragraph result.
 		await page.keyboard.press( 'Enter' ); // Insert paragraph.
-		await page.keyboard.type( 'First col' );
+		await page.keyboard.type( '1st col' );
```

Comment:
Why should this change?

Reply:
This one technically doesn't need to change for the test to pass today. Just the second one was causing issues. But I figured it would make sense to be consistent about this wording.

Reply:
I guess I don't understand why the second one is changing either. You've just changed `Second` to `2nd` everywhere? What difference does that make?

Reply:
It's a not-so-great fix, but it solves the problem that @tellthemachines [noted above](https://github.com/WordPress/gutenberg/pull/14961#issuecomment-508614214), and gets the tests to pass. 

Essentially, the text "Second column" was dropping to its own new line when the additional padding was added, breaking the number of `ArrowUps` expected to get up to the toolbar. Shortening that text to "2nd column" prevents the line break, and allows the tests to keep passing. 

I'm definitely up for a more future-proof solution, but not sure what that would be.

Reply:
Ah, I see now what you mean. In that case, please, please, inline comment the test so we know why it changed and would fail otherwise. :)

Reply:
> please, please, inline comment the test so we know why it changed and would fail otherwise. :)

Great idea! Added in 1b9d6bb.